operations/restrictions.md

```diff
@@ -260,119 +216,322 @@ Adds new restrictions with the specified conditions. Care is needed to specify `
+| `Restrictions` | array of [Restriction data](restrictions.md#restriction-data) | required, max 1000 items | Parameters of restrictions. |
+
+#### Restriction data
 
-#### Restriction parameters
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Identifier` | string | optional | Identifier of the restriction within the transaction. |
 | `ExternalIdentifier` | string | optional | External identifier of the restriction. |
-| `Conditions` | [Restriction Conditions](#restriction-conditions) | required | The conditions or rules that must be met by a reservation for the restriction to apply. |
-| `Exceptions` | [Restriction Exceptions](#restriction-exceptions) | optional | The rules that prevent the restriction from applying to a reservation, even when all conditions have been met. |
+| `Conditions` | [Restriction condition data](restrictions.md#restriction-condition-data) | required | The conditions or rules that must be met by a reservation for the restriction to apply. |
+| `Exceptions` | [Restriction exception data](restrictions.md#restriction-exception-data) | optional | The rules that prevent the restriction from applying to a reservation, even when all conditions have been met. |
+
+#### Restriction condition data
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Type` | [Restriction type](restrictions.md#restriction-type) | required | Restriction type. |
+| `ExactRateId` | string | optional | Unique identifier of the exact `Rate` to which the restriction applies. |
+| `BaseRateId` | string | optional | Unique identifier of the base `Rate` to which the restriction applies. |
+| `RateGroupId` | string | optional | Unique identifier of the `Rate group` to which the restriction applies. |
+| `ResourceCategoryId` | string | optional | Unique identifier of the `Resource category` to which the restriction applies. |
+| `ResourceCategoryType` | [Resource category type](restrictions.md#resource-category-type) | optional | Name of the `Resource category type` to which the restriction applies. |
+| `StartUtc` | string | optional | Start date of the time interval for which the restriction conditions should be applied. This must be in UTC timezone in ISO 8601 format - see [Datetimes](../guidelines/serialization.md#datetimes). |
+| `EndUtc` | string | optional | End date of the time interval for which the restriction conditions should be applied. This must be in UTC timezone in ISO 8601 format - see [Datetimes](../guidelines/serialization.md#datetimes). |
+| `Days` | array of string | optional | The restricted days of week. Defaults to all days when not provided. Ignored when the service uses a time unit longer than a day. |
+| `Hours` | [Hours parameters](restrictions.md#hours-parameters) | optional | The restricted hours. Defaults to all hours when not provided. |
+
+#### Hours parameters
```

Comment:
Hours list was not documented before and now it appears twice in the docs. As per discussion, they can be unified into a single one.

Reply:
Removed